courses/machine_learning/deepdive2/launching_into_ml/solutions/explore_data.ipynb

```diff
@@ -0,0 +1,1980 @@
+{
```

Comment:
<p>Backslashes are redundant. The style checker could be useful here.</p><p><a href="http://pep8online.com/checkresult" target="_blank">http://pep8online.com/checkresult</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/654/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p>